google/ggrc-core
# GGRC-6159 Audit captains were removed from CC list in Issue Tracker

# Issue description

Deleting records from IssueTracker, that were configured not from GGRC system

# Steps to test the changes
Run integration tests
Create Assessment with IssueTracker - ON. CCs synchronization for assessment should be only on periodic task (combination users from ggrc system - audit captains and assignees, and users from issue tracker). Synchronization on PUT signal for assessment was removed. CCs records stored only in databases on PUT signal for Audit and Assignees independently, without IssueTracker synchronization.

# Solution description
Remove CCs synchronization on Assignee PUT signal
Update CCs combination logic with union from IssueTracker on periodic task
Update "the need of synchronization" validation for CCs (validate by "subset")

# Sanity checklist



# PR Review checklist
